src/content/docs/en/guides/deploy/fleek.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Deploy your Astro Site to Fleek
+description: How to deploy your Astro site to the web on Fleek.
+type: deploy
+i18nReady: true
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
+
+You can use [Fleek](http://fleek.xyz/) to deploy an Astro site to their edge-optimized decentralized network.
+
+This guide gives a complete walkthrough of deploying your Astro site to Fleek using the Fleek UI and CLI.
+
+
+## Project configuration
+
+In your `astro.config.mjs`, you can configure your project to deploy to Fleek by setting the `output` to `static`:
+
+```js title="astro.config.mjs" ins={5}
+import { defineConfig } from 'astro/config';
+
+// https://astro.build/config
+export default defineConfig({
+    output:'static',
+});
+```
+
+## How to deploy
+
+You can deploy to Fleek through the website UI or using Fleek’s CLI (command line interface).
+
+
+### Platform UI Deployment
+
+<Steps>
+1. Create a [Fleek](https://app.fleek.xyz) account
+
+2. Push your code to your online Git repository (GitHub).
+
+3. Import your project into Fleek.
+
+4. Fleek will automatically detect Astro and then you can configure the correct settings.
+
+5. Your application is deployed! (e.g. [astro-fleek.on-fleek.app/](https://astro-fleek.on-fleek.app/))
```

Comment:
None of the Deploy guides provide exemplar sites in their procedure. Best not to make readers click upon links that won't help their tasks
```suggestion
5. Your application is deployed!
```

Reply:
Sure thing!